builtins/amp-layout.md

```diff
@@ -25,26 +25,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# <a name="amp-layout"></a> `amp-layout`
+#amp-layout <a name="amp-layout"></a>
```

Comment:
Missing a space between `#` and "amp-layout".

Reply:
Thanks for catching!